ios/Example/Tests/TinkTest.swift

```diff
@@ -9,9 +9,9 @@ import Foundation
 import XCTest
 
 final class TinkTests: XCTestCase {
```

Comment:
I think we should delete this since its represented in TrifleTests (I probably named it incorrectly when I oepned the PR)